integration/client-cli/admin-cli/src/main/java/org/keycloak/client/admin/cli/commands/AbstractRequestCmd.java

```diff
@@ -103,100 +104,57 @@ public abstract class AbstractRequestCmd extends AbstractAuthOptionsCmd {
 
     String httpVerb;
 
-    Headers headers = new Headers();
+    @Option(names = {"-h", "--headers"}, description = "Set request header NAME to VALUE")
```

Comment:
```suggestion
    @Option(names = {"-h", "--header"}, description = "Set request header NAME to VALUE")
```
I think it should be like this in order to reflect the old behavior.